src/components/common/EstimateParser.tsx

```diff
@@ -6,7 +6,7 @@ interface IProps {
 
 function EstimateParser({seconds}: IProps) {
   const parsedTimer = new window.Date(seconds * 1000).toISOString().substr(11, 8);
-  return <React.Fragment>{parsedTimer}</React.Fragment>;
+  return <>{parsedTimer}</>;
```

Comment:
Can we please continue using `React.Fragment` here? I prefer that syntax way more :D

Reply:
Ah okay, I like the `<>` syntax, but sure no worries!